samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -36,14 +38,27 @@ const SitecorePage = ({ notFound, layoutData, componentProps }: SitecorePageProp
 
 // This function gets called at build and export time to determine
 // pages for SSG ("paths", as tokenized array).
-export const getStaticPaths: GetStaticPaths = async () => {
+export const getStaticPaths: GetStaticPaths = async (context) => {
   // Fallback, along with revalidate in getStaticProps (below),
   // enables Incremental Static Regeneration. This allows us to
   // leave certain (or all) paths empty if desired and static pages
   // will be generated on request.
   // See https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration
-  //
-  // Ultimately, this is where we'll also be able to request a "sitemap" from Sitecore.
+
+  // use graphQLSitemapService.fetchExportSitemap(context.defaultLocale, ROOT_ITEM)
```

Comment:
Should this note be further down next to the `fetchSSGSitemap` call? Part of me wants to just leave this export comment out and leave it up to the export documentation...

Reply:
Agree, can remove - will add it to doc for export